### PR TITLE
CP-1453 Upgrade to coverage 0.7.3 to fix coverage issues

### PR DESCRIPTION
**Pull request by evanweible-wf**
## Issue

Coverage broke with Dart SDK 1.14. The coverage package released version 0.7.3 to address this issue, although it was only compatible with Dart SDK 1.15. Now that 1.15 is out, we need to update and make sure coverage is working.

## Changes

Upgrade to coverage 0.7.3.

## Testing
- [ ] Ensure all tests pass
- [ ] Run coverage in a project with VM and browser tests and ensure that it is correctly collected.

## Code Review
@trentgrover-wf 
@maxwellpeterson-wf 
@dustinlessard-wf 
@jayudey-wf

### CODE REVIEW

**Comment by dustinlessard-wf** [comment]
+1 !

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
+1
@jayudey-wf ready for merge

File: pubspec.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ homepage: https://github.com/Workiva/dart_dev
 dependencies:
   ansicolor: "^0.0.9"
   args: "^0.13.0"
-  coverage: "^0.7.2"
+  coverage: "^0.7.3"
```
Comment (jayudey-wf) [inline pubspec.yaml]: we should update the readme to document the change in the recommended dependencies https://github.com/Workiva/dart_dev#install-dart_dev

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - verified that tests were still passing w/ dart sdk 1.14.0 and 1.15.0
 - installed into w_flux and verified that coverage was still able to be collected
- [ ] Unit test created/updated
- [ ] All unit tests pass

Merging into master.

**Pull request merged**